<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigDebug</class>
 <widget class="QWidget" name="frmConfigDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTextEdit" name="txtMain">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameRight">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labPortName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>端口名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QComboBox" name="cboxKeyColor1"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="labDeviceName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>控制器名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QTextEdit" name="txtKeyValue1"/>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QTextEdit" name="txtKeyValue2"/>
      </item>
      <item row="12" column="0">
       <widget class="QCheckBox" name="ckSendData">
        <property name="text">
         <string>发送数据</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QCheckBox" name="ckReceiveInfo">
        <property name="text">
         <string>解析数据</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="cboxPortName"/>
      </item>
      <item row="13" column="1">
       <widget class="QCheckBox" name="ckPause">
        <property name="text">
         <string>暂停显示</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QComboBox" name="cboxKeyColor2"/>
      </item>
      <item row="12" column="1">
       <widget class="QCheckBox" name="ckReceiveData">
        <property name="text">
         <string>接收数据</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QComboBox" name="cboxDeviceName"/>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="btnClearData">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>清空数据</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="labKeyValue1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>关键字过滤1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="labKeyValue2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>关键字过滤2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="btnReadValueAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>读取所有</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QCheckBox" name="ckOther">
        <property name="text">
         <string>其他数据</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QCheckBox" name="ckDebug">
        <property name="text">
         <string>打印输出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtMain</tabstop>
  <tabstop>cboxPortName</tabstop>
  <tabstop>cboxDeviceName</tabstop>
  <tabstop>cboxKeyColor1</tabstop>
  <tabstop>txtKeyValue1</tabstop>
  <tabstop>cboxKeyColor2</tabstop>
  <tabstop>txtKeyValue2</tabstop>
  <tabstop>btnClearData</tabstop>
  <tabstop>btnReadValueAll</tabstop>
  <tabstop>ckSendData</tabstop>
  <tabstop>ckReceiveData</tabstop>
  <tabstop>ckReceiveInfo</tabstop>
  <tabstop>ckPause</tabstop>
  <tabstop>ckOther</tabstop>
  <tabstop>ckDebug</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
